package.json

```diff
@@ -58,7 +58,7 @@
   "dependencies": {
     "dashjs": "^2.2.0",
     "global": "^4.3.2",
-    "video.js": "^6 || ^7"
+    "video.js": "^5.18.0 || ^6 || ^7"
```

Comment:
the previous 5.x value was 5.18: https://github.com/videojs/videojs-contrib-dash/pull/276/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L59